__tests__/diff-snapshot.spec.js

```diff
@@ -219,12 +223,38 @@ describe('diff-snapshot', () => {
       expect(mockWriteFileSync).toHaveBeenCalledTimes(1);
     });
 
+    it('should write a received image if the test fails and storeReceivedOnFailure = true', () => {
```

Comment:
Can you add another test for when `storeReceivedOnFailure` is false

Reply:
I added an explicit test case for this situation. However, this was already tested implicitly since false is the default. I did change the implementation a little bit so that it also does not return the `receivedSnapshotPath` if `storeReceivedOnFailure` is set to false.